user/preprocess/input.md

```diff
@@ -51,7 +51,8 @@ The CB-Geo MPM code uses a `JSON` file for input configuration.
       "uuid": "usf-axial-loading-5cb93af",
       "step" : 5
     },
-    "gravity": [0.0, 0.0, -9.81]
+    "gravity": [0.0, 0.0, -9.81],
+    "pressure_smoothing" : false
```

Comment:
We don't have pressure smoothing

Reply:
Let me delete from it now, let #6 handles it